ExtractMarkupTool.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+from PIL import Image, ImageDraw, ImageChops, ImageOps
+
+
+class DataSetCreationTool(object):
+    CSV_COLUMN_INDEX_IMAGE_NAME = 0
+    CSV_COLUMN_INDEX_REGION_SHAPE_ATTRIBUTES = 5
+
+    def __init__(self, csv_path):
+        self.csv_path = csv_path
+
+    # Imports the .csv file extracted from VGG Image Annotator. Works for sure with only one region per frame.
+    # Unfortunately, there are a lots of syntaxes that I could not use without deleting them to make the data usable
+    def import_csv(self):
+        data = pd.read_csv(self.csv_path)
+
+        # Deletes images that are not labelled in the .csv file
+        data = data[data.values[:, CSV_COLUMN_INDEX_REGION_SHAPE_ATTRIBUTES] != "{}"]
+
+        data2 = pd.DataFrame(data.values[:, CSV_COLUMN_INDEX_REGION_SHAPE_ATTRIBUTES],
+                             data.values[:, CSV_COLUMN_INDEX_IMAGE_NAME])
+
+        file_names = [data.values[:, CSV_COLUMN_INDEX_IMAGE_NAME]]
+        x = []
+        y = []
+
+        # Making the format readable
+        for row in range(len(data2)):
+            file_names.append(data2.values[row, 0].split(",")[0])
+            x.append(data2.values[row, 0].split("all_points")[1])
+            y.append(data2.values[row, 0].split("all_points")[2])
+
+        xList = []
+        yList = []
+
+        # Making a list of x coordinates for each images
+        for element in range(len(x)):
+            x[element] = x[element].replace('_x":', "")
+            x[element] = x[element].replace(',"', "")
+            x[element] = x[element].replace('[', "")
+            x[element] = x[element].replace(']', "")
+
+        # Splitting them
+        for element in x:
+            xList.append(element.split(",", ))
+
+        # Converting them to integers
+        for element in xList:
+            for value in range(len(element)):
+                element[value] = int(element[value])
+
+        # Same for the y coordinates
+        for element in range(len(y)):

```

Comment:
You are doing the same thing twice, for x and y. How about create a specific method for it?